### PR TITLE
Add Fedora installation info

### PR DESCRIPTION
**Pull request by tim77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/ajeetdsou
 If you want the interactive fuzzy selection feature, you will also need to install
 [`fzf`](https://github.com/junegunn/fzf.git).
 
+#### Packaging status
+
+Fedora/CentOS [COPR](https://copr.fedorainfracloud.org/coprs/atim/zoxide/): `sudo dnf copr enable atim/zoxide -y && sudo dnf install zoxide`
```
Comment (cole-h) [inline README.md:67]: I would personally rather see this split into separate lines, as well as drop the "automatic"-ness of the `-y` flag:

```shell
dnf copr enable atim/zoxide
dnf install zoxide
```

Additionally, if the user needs to run this as root, they should do it themselves and not need to be told by us.

**Comment by ajeetdsouza** [comment]
Looks like `zoxide` has been added to Fedora's official repos, so I'm closing this. Thank you for your work!

**Comment by tim77** [comment]
> Looks like `zoxide` has been added to Fedora's official repos, so I'm closing this. Thank you for your work!

Only for F32. And even without shell completion. It's up on you but if you want in future not stale and always fresh and real `zoxide` for all current Fedora releases and CentOS, you know where you can find it. 😉

**Comment by ajeetdsouza** [comment]
@tim77 I've added instructions for CentOS in https://github.com/ajeetdsouza/zoxide/commit/27389cee25779c31014b3c28bfce4d2a3d308219. Thank you for maintaining zoxide!

**Pull request closed without merging**